WooCommerce/Classes/Authentication/Epilogue/EmptyStoresTableViewCell.swift

```diff
@@ -68,7 +77,9 @@ private extension EmptyStoresTableViewCell {
             comment: "Link on the store picker for users who signed in with Apple to close their WordPress.com account."
         )
         static let legend =
-            NSLocalizedString("Your account isn’t connected to any WooCommerce stores.",
+            NSLocalizedString("Create your first store.",
```

Comment:
Super nit: Do we need a full stop here? I don't see one on the Figma design.

Reply:
Good catch! Removed in b60842af0646be698ff14d675e463d6130c27656.